modules/core/include/opencv2/core/hal/intrin_vsx.hpp

```diff
@@ -1051,6 +1051,15 @@ inline v_float64x2 v_cvt_f64(const v_float32x4& a)
 inline v_float64x2 v_cvt_f64_high(const v_float32x4& a)
 { return v_float64x2(vec_cvfo(vec_mergel(a.val, a.val))); }
 
+// The altivec intrinsic is missing for this 2.06 insn
+inline v_float64x2 v_cvt_f64(const v_int64x2& a)
+{
+vec_double2 out;
+
+__asm__ ("xvcvsxddp %x0,%x1" : "=wa"(out) : "wa"(a.val));
```

Comment:
too late from me but any use of operator modifier `x` in inline asm that effect on clang should guard it by definition `CV_COMPILER_VSX_BROKEN_ASM` since a wide range of clang versions don't support it,
also `xvcvsxddp` already been covered by `vec_ctd` for both compilers in `vsx_utils.h`, I the updated issue #15506 to contain clang build failure logs and it will be resolved by #15510